lib/transcoder/model/video.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Transcoder.Model.Video do
+  use TranscoderWeb, :verified_routes
+
+  defstruct [:filename, resolution: :original]
+
+  def path(video) do
+    Path.join([:code.priv_dir(:transcoder), "static", "uploads", video.filename])
+  end
```

Comment:
If we want to reliably generate paths, we might need to also include the resolution, right?

Reply:
Reorganized logic to generate filename in single place.